GitExtensions.sln

```diff
@@ -356,8 +356,8 @@ Global
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Release|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Since I first cloned the GitExtensions repository, I got this message from Visual Studio every time I press F5:

![Visual Studio debug message](https://user-images.githubusercontent.com/24460532/47081109-06c6d500-d20a-11e8-8be1-b5258681b159.png)

Solution was the mentioned change. I didn't commit this change before. At such operations like rebase, I had always either stash or reset the change.

Does this change have a negative effect? Should I open an issue for this? I wonder why nobody else seems to have this problem.

Reply:
This must stay as is.
This was changed in #5269 in https://github.com/gitextensions/gitextensions/pull/5269/commits/f01dad36497fe16e05a211d8a09a3c0dfc604afa

Reply:
When I had this dialog I just selected "Continue Debugging (Don't Ask Again)"

Reply:
Rationale here is that the debug build does a lot of internal validation that hurts performance considerably when debugging. Before we brought this in as a submodule, it was a NuGet package and used in release mode.

Maybe we should add a comment in the `.sln` file about this.